src/transform/CollapseTable.js

```diff
@@ -309,6 +309,37 @@ const newCaptionFragment = (document, title, headerText) => {
   return fragment
 }
 
+/**
+ * @param {!Node} nodeToReplace
+ * @param {!Node} replacementNode
+ * @return {void}
+ */
+const replaceNodeInContentBlock = (nodeToReplace, replacementNode) => {
+  if (!nodeToReplace || !replacementNode) {
+    return
+  }
+  let lastAncestorBeforeContentBlock = nodeToReplace
+  let contentBlock = nodeToReplace.parentNode
+  if (!contentBlock) {
+    return
+  }
+  let foundContentBlock = false
+  while (contentBlock) {
+    if (contentBlock.classList && contentBlock.classList.contains('content_block')) {
+      foundContentBlock = true
+      break
+    }
+    lastAncestorBeforeContentBlock = contentBlock
```

Comment:
Since this is going upwards, isn't `lastAncestorBeforeContentBlock` really `childOfContentBlock`?

Reply:
Updated